src/vs/workbench/contrib/tasks/electron-browser/task.contribution.ts

```diff
@@ -1973,6 +1973,9 @@ class TaskService extends Disposable implements ITaskService {
 			if ((entries.length === 0) && defaultEntry) {
 				entries.push(defaultEntry);
 			}
+			else if (selectedEntry) {
```

Comment:
Rather than using the selectedEntry, let's add a new paramter to `showQuickPick` for additional `TaskQuickPickEntry` items.